plugin/pkg/auth/authorizer/rbac/rbac.go

```diff
@@ -121,6 +121,8 @@ func (r *RBACAuthorizer) Authorize(requestAttributes authorizer.Attributes) (aut
 	reason := ""
```

Comment:
nit: `reason := "no RBAC policy matched"` instead of the `else`

Reply:
I don't like setting and then overwriting :)